package.json

```diff
@@ -39,7 +39,7 @@
     "mobile-cli-lib": "https://github.com/telerik/mobile-cli-lib/tarball/master",
     "mocha": "3.5.3",
     "mocha-typescript": "1.1.8",
-    "nativescript": "https://github.com/NativeScript/nativescript-cli/tarball/master",
+    "nativescript": "https://github.com/NativeScript/nativescript-cli/tarball/kerezov/cloud-deploy-run",
```

Comment:
Not needed :smile:

Reply:
@Fatme it's needed for successful transpilation - I've marked the commit with `[delete]`